SharpXMPP.Shared/XMPP/JID.cs

```diff
@@ -12,33 +12,35 @@ public JID()
         {
 
         }
-        
+
         public JID(string jid)
         {
-            var domainWithResource = string.Empty;
-            var jidParts = jid.Split(new[] { '@' }, StringSplitOptions.RemoveEmptyEntries);
+            var bareJid = string.Empty;
+            var jidParts = jid.Split(new[] {'/'}, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
You know who'll be the next user who'll break the parsing? A man named `user/@` or something like that :)

Could you please use [`String.Split(Char[], Int32)`](https://docs.microsoft.com/en-us/dotnet/api/system.string.split?view=netstandard-2.0#System_String_Split_System_Char___System_Int32_) overload here, since we don't expect more than two JID parts anyway?